src/python/dxpy/scripts/dx.py

```diff
@@ -3758,25 +3758,29 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 register_parser(parser_cat, categories='data')
 
 parser_head = subparsers.add_parser('head',
-                                    help='Print part of a file or gtable',
-                                    description='Print the first part of a file or a gtable.  By default, prints the first 10 lines or rows, respectively.  Additional query parameters can be provided in the case of gtables.  The output for gtables is formatted for human-readability; to print rows in a machine-readable format, see "dx export tsv".',
+                                    help='Print part of a file',
+                                    description='Print the first part of a file.  By default, prints the first 10 lines.',
                                     parents=[no_color_arg, env_args],
                                     prog='dx head')
-parser_head.add_argument('-n', '--lines', type=int, metavar='N', help='Print the first N lines or rows (default 10)',
+parser_head.add_argument('-n', '--lines', type=int, metavar='N', help='Print the first N lines (default 10)',
                          default=10)
 head_gtable_args = parser_head.add_argument_group(title='GTable-specific options')
```

Comment:
FWIW after all the options in this category were marked as help-suppressed, this section header is no longer displayed.